src/test/java/edu/kis/powp/jobs2d/TestJobs2dApp.java

```diff
@@ -127,6 +117,16 @@ private static void setupLogger(Application application) {
 		application.addComponentMenuElement(Logger.class, "OFF logging", (ActionEvent e) -> logger.setLevel(Level.OFF));
 	}
 
+	private static void setUpExtensions(){
+		Job2dDriver driver = new LoggerDriver();
+		ExtensionFeature.addExtensionDriver("Logger driver", driver);
+
+		DrawPanelController drawerController = DrawerFeature.getDrawerController();
+		driver = new DeviceUsageDecorator(new LineDriverAdapter(drawerController, LineFactory.getBasicLine(), "basic"), DeviceUsageFeature.getDeviceUsageManager());
+		ExtensionFeature.addExtensionDriver("Line Simulator with Device Usage",driver);
```

Comment:
It would be best to redesign Device Usage so it wouldn't use internal LineDriverAdapter anymore.